bundles/org.eclipse.swt/Eclipse SWT Custom Widgets/common/org/eclipse/swt/custom/CTabFolderRenderer.java

```diff
@@ -1402,6 +1395,35 @@ void drawSelected(int itemIndex, GC gc, Rectangle bounds, int state ) {
 				gc.setForeground(borderColor);
 				gc.drawPolyline(shape);
 
+				// draw highlight marker of selected tab
+				if (parent.selectionHighlightBarThickness > 0 && parent.simple) {
```

Comment:
Shouldn't we still draw the highlight line before the border/outline? Otherwise the highlight will be drawn on top of it, thus hiding the outline.